source/configure/environment-configuration-settings.rst

```diff
@@ -3643,6 +3644,95 @@ Allow untrusted internal connections
 |   but not ``10.0.16.20``.                                                                                                                     |
 +-----------------------------------------------+-----------------------------------------------------------------------------------------------+
 
+Mobile security
```

Comment:
@cwarnermm we need to include that these settings are only available for Enterprise licensed servers.

Reply:
Thanks, @marianunez! The ``.. include:: ../_static/badges/ent-only.rst`` code directly under the section title displays the Enterprise only badge.

Reply:
Ah thanks @cwarnermm! Wasn't familiar with those badges 👍